### DISCUSSION TITLE
Tribler docker compose version

### DISCUSSION THREAD

**Discussion by tipitao** [Tribler/tribler#8232] [Q&A]
Hi everybody. Hi have DYS nas with Open mediavault 7.x  (x64). I wish deploy Trible with docker compose file but i dont know if thery a version avaiable.

Regards
Sorry my poor enghish

**Comment by qstokkink** [comment]
We don't have instructions for Docker Compose right now. However, we do have instructions for "normal" Docker:

https://tribler.readthedocs.io/en/latest/basics/docker.html

**Comment by tipitao** [comment]
Ok thanks
